dataland-keycloak/dataland_theme/login/src/passwordStrength.ts

```diff
@@ -0,0 +1,87 @@
+import { zxcvbn, zxcvbnOptions, debounce } from '@zxcvbn-ts/core'
+import zxcvbnCommonPackage from '@zxcvbn-ts/language-common'
+import zxcvbnEnPackage from '@zxcvbn-ts/language-en'
+import { InputField } from "./utils/InputField";
+
+zxcvbnOptions.setOptions({
+    translations: zxcvbnEnPackage.translations,
+    graphs: zxcvbnCommonPackage.adjacencyGraphs,
+    dictionary: {
+        ...zxcvbnCommonPackage.dictionary,
+        ...zxcvbnEnPackage.dictionary,
+    },
+})
+
+const passwordFieldWrappingDiv = document.querySelector<HTMLDivElement>('div[data-role=password-primary]')!
+const passwordField = new InputField(passwordFieldWrappingDiv)
+
+const confirmPasswordFieldWrappingDiv = document.querySelector<HTMLDivElement>('div[data-role=password-confirm]')!
+const confirmPasswordField = new InputField(confirmPasswordFieldWrappingDiv)
+
+const form = document.querySelector<HTMLInputElement>('form')!
+const passwordStrengthIndicatorInner = document.querySelector<HTMLInputElement>('#password-strength-indicator > div')!
+
+function updatePasswordStrengthIndicator(levelCss: string) {
+    passwordStrengthIndicatorInner.classList.forEach((cls) => {
+        if (cls.startsWith("d-password-strength")) {
+            passwordStrengthIndicatorInner.classList.remove(cls)
```

Comment:
Hier removen wir sachen aus einem Array, während wir darüber iteriren. Zumindest in Java macht das Probleme wenn man das Array, über das man gerade iteriert, verändert. 

Was denkst du?

Reply:
Vielleicht besser: Wir haben eine const Liste mit allen möglichen Klassen, machen ein 
```
passwordStrengthIndicatorInner.classList.remove(allPossibleClasses)
```
oder?

Reply:
Das ist wahrscheinlich sicherer. Ändere ich ab

Reply:
ok